### ISSUE TITLE
Installing 2.6.3 and missing database?

### ISSUE THREAD

**Issue by jsiree** [Esri/geoportal-server-catalog#323]
Hi,
I have done a fresh installation to windows server 2012r2 and the geoportal application does not start in Tomcat. There was no instruction to set up database for 2.6.3 version or have I missed something? In previous version 1.2.9 there are instructions to set up a database. Is that database valid for this version 2.6.3?

Thanks!

**Comment by mhogeweg** [comment]
hi, you have not missed anything. Geoportal 2.6.3 does not use a relational database. It does use elasticsearch. Installation instructions for 2.6.3 are [here](https://github.com/Esri/geoportal-server-catalog/wiki/Installing-Geoportal-Server-2.6.3).

**Comment by jsiree** [comment]
Thanks, this is my first time installing this Geoportal. I did not realize that elasticsearch is used to store metadata. Will try Kaizen when installation is success. Nice to know! Should have read more about these components. But elasticsearch is installed.

And also my server is not connected to internet. I have followed installation instructions and done configuring for disconnected environments.

When I look at the logs there is errors about springframework xml schema definitions. Because I am not connected to internet, I cannot read those. Is there a workaround for this? Could this be the problem with my installation?
